public/index.html

```diff
@@ -15,6 +15,7 @@
     <!-- Custom styles for this template -->
     <link href="assets/css/main.css" rel="stylesheet">
 	<link rel="stylesheet" href="assets/css/font-awesome.min.css">
+	<link rel="shortcut icon" href="assets/css/font-awesome.min.css">
```

Comment:
@wingchunkf what is this dup link tag doing?

Reply:
That's not mine but I should have noticed it. Will remove now.